db/scripts/seed_database.js

```diff
@@ -27,8 +27,8 @@ async function seedDatabase() {
 
   await db.result(
     `INSERT INTO scheduled_calls_table
-    (patient_name, recipient_email, recipient_name, call_time, call_id, provider, ward_id, call_password)
-    VALUES ('Alice', 'bob@example.com', 'Bob', CURRENT_TIMESTAMP, '123', 'whereby', $1, 'password')`,
+    (patient_name, recipient_email, recipient_name, call_time, call_id, provider, ward_id, call_password, status)
+    VALUES ('Alice', 'bob@example.com', 'Bob', CURRENT_TIMESTAMP + interval '1 hour', '123', 'whereby', $1, 'password', 'scheduled')`,
```

Comment:
I think we should keep the timestamp as the current time so that we have a visit that is starting 'now' in case the scheduled visit filtering code ends up affecting the visits tests again

Reply:
Hmm.. not sure about that. My thinking is that the visit isn't starting 'now' from the PoV of the tests, but a few seconds/minutes ago. I think what we should really have is a set of visits defined to match each state so that we can test them all.
Definitely room for improvement though. Perhaps we could improve the seed data while doing one of the other metric tickets?

Reply:
A visit that started a few seconds ago represents a visit starting now in the real world (staff start visits well after they are meant to start which is why we keep visits around so long)

Reply:
I'm keen to unblock this PR though so happy to merge and fix the missing test coverage around retrieveVisits later